modules/caddytls/connpolicy.go

```diff
@@ -587,6 +609,12 @@ type ConnectionMatcher interface {
 	Match(*tls.ClientHelloInfo) bool
 }
 
+// LeafCertificateLoader is a type that loads the trusted leaf certificates
+// for the tls.client_auth.leaf module
```

Comment:
```suggestion
// for the tls.leaf_cert_loader modules
```

I think this is a little more accurate (but I knew what you meant)